examples/main/layer-examples.js

```diff
@@ -194,7 +194,7 @@ const ScatterplotLayer64Example = {
 // perf test examples
 const ScatterplotLayerPerfExample = ({
   layer: ScatterplotLayer,
-  getData: dataSamples.getPoints100K,
+  getData: dataSamples.getPoints1M,
```

Comment:
Any chance you can restore the 10M 32 bit and 1M 64 bit as well? They were lost in my refactor?